src/test/kotlin/jces1209/AwsVus.kt

```diff
@@ -22,7 +26,10 @@ import java.time.Duration
 import java.util.UUID
 import java.util.concurrent.CompletableFuture
 
-class AwsVus(duration: Duration) : VirtualUsersSource {
+class AwsVus(
```

Comment:
Maybe we should start thinking about AwsVus builder? 😃

Reply:
Test authors want to compare different `BenchmarkQuality` instances at the same time, so they get a builder for top-level usage. `AwsVus` is not used directly, because it's encapsulated by `SlowAndMeaningful`.
But I had the same thought initially too 🙂